index.html

```diff
@@ -14,28 +14,50 @@
+            <li><img src="Images/Union.png" alt=""></li>
+        </ul>
     </header>
 
-    <section class="headline">
-        <h1 class="intro">Hey There. I'm Daniel</h1>
-        <h2 class="developer">I'm a Software developer</h2>
-        <p class="abilities">I can help you build a product , feature or website Look through some of my work and
-            experience!
-            If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <ul class="icons">
-            <li class="github"><a href=""><i class="fa-brands fa-github"></i></a></li>
-            <li class="linkedin"><a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
-            <li class=angelist><a href=""><i class="fab fa-angellist"></i></a></li>
-            <li class="twitter"><a href=""><i class="fa-brands fa-twitter"></i></a></i></li>
-            <li class="facebook"><a href=""></a><i class="fa-brands fa-facebook-f"></i></li>
-        </ul>
+    <section class= "cont-alls">
+        <span class= "headline">
+            <ul class="icons">
+                <li class="github" id="mines"><a href=""><img src="images/github.png" alt="" id="centered-icons"></a></li>
+                <li class="linkedin" id="mines"><a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+                <li class=angelist id="mines"><a href=""><img src="images/angel list.png" alt="" id="centered-icons"></a></li>
+                <li class="twitter" id="mines"><a href=""><img src="images/twitter.png" alt="" id="centered-icons"></a></li>
+                <li class="facebook" id="mines"><a href=""></a><i class="fa-brands fa-facebook-f"></i></li>
+            </ul>
+                        
+                            <div id="write">
+                               <div><h1 class="intro" id="first">Hey There. I'm Daniel</h1></div> 
+                           <div><h2 class="developer" id="first">I'm a Software developer</h2></div> 
+                            <div><p class="abilities" id="first">I can help you build a product , feature or website Look through some of my work and
+                                experience!
+                                If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                       </div>
+                            </div>
+        </span>
+        
+                    
+               
+                
+       
     </section>
 
     <section class="recent">
-        <h2 class="recent-item">My Recent Works</h2>
+       <div id="recent"> <h2 class="recent-item">My Recent Works</h2>
         <div class="line"></div>
+        <img src="images/Vector-4.png"  alt="a line"  class="divide"/>
+    </div>
+    <div id="desktop-layout">
         <div class="yellowboard">
-            <img src="images/bigyellow.png" width=327 height=237 alt="">
+            <img src="images/bigyellow.png" width=327 height=237 alt="" id="yellow">
```

Comment:
- [ ] Please kindly make sure you stick to the design as much as possible , try position elements in the work card perfectly in the card to match figma design and that will make your project looks good.you got this 👍 

Figma design(mobile):
![image](https://user-images.githubusercontent.com/90524466/186719096-5913132b-10cf-4428-a4ea-fe03ef5390b6.png)

Your design(mobile):
![image](https://user-images.githubusercontent.com/90524466/186719202-56f335b9-5f5f-4593-8429-5d953f7d0e64.png)

Your design(desktop):
![image](https://user-images.githubusercontent.com/90524466/186719659-4038558f-0657-4052-b052-e1d2905fc162.png)

Figma design(desktop):
![image](https://user-images.githubusercontent.com/90524466/186719788-ffdee809-daa2-4d71-b64d-f1dfaa76d28c.png)

Reply:
I have fixed the following problems, staged, committed and pushed them. Thank you for pointing it out